### DISCUSSION TITLE
Closing terminal/console shuts down the artisan command

### DISCUSSION THREAD

**Discussion by JPeterson2015** [spatie/twitter-streaming-api#36] [Q&A]
When I run the artisan command to start listening for hashtags and then close my servers console the artisan command stops running...

Is there a proper way to do this?

**Comment by felixdorn** [comment]
Yes!

You can use [nohup](https://linux.die.net/man/1/nohup) for that.

So your command would look like this:

```
nohup php artisan my:command > /dev/null 2>&1 &
```
You may replace the `&1` with a file path to save error logs if you print some.

You may also use `screen` if the process is interactive and you need to come back to it.

If you're looking for a more robust solution you can daemonize the process.
